Formula/g/gensio.rb

```diff
@@ -0,0 +1,47 @@
+class Gensio < Formula
+  desc "Stream I/O Library"
+  homepage "https://github.com/cminyard/gensio"
+  url "https://downloads.sourceforge.net/project/ser2net/ser2net/gensio-2.7.5.tar.gz"
+  sha256 "2e55ac602062b0807a97cf3d9f0b944bc9667d14b76d67476c579d0615ea574b"
+  license all_of: ["LGPL-2.1-only", "GPL-2.0-only", "Apache-2.0"]
+
+  depends_on "go" => [:build]
+  depends_on "pkg-config" => [:build]
+  depends_on "swig" => [:build]
+  depends_on "glib"
+  depends_on "openssl@3"
+  depends_on "portaudio"
+  depends_on "python@3.11"
+
+  def python3
+    "python3.11"
+  end
+
+  def install
+    # https://rubydoc.brew.sh/Formula.html#std_configure_args-instance_method
+    system "./configure", *std_configure_args.reject { |s| s["--disable-debug"] },
+                          "--disable-silent-rules",
+                          "--with-pythoninstall=#{lib}/gensio-python",
+                          "--with-link-ssl-with-main",
+                          "--with-homebrew-prefix=#{HOMEBREW_PREFIX}",
```

Comment:
These seem very unusual, what do they do?

Reply:
> There is an automated test for this, but it won't work cross. 

Cross what?

> The automated test I mention above needs to know where to load the openssl dynamic library from.

`pkg-config --libs openssl` would be the standard one. Using `brew config` for this is just waiting for trouble since that's not a stable API.

Reply:
> > There is an automated test for this, but it won't work cross.
> 
> Cross what?

Oh, sorry, cross compile.

> 
> > The automated test I mention above needs to know where to load the openssl dynamic library from.
> 
> `pkg-config --libs openssl` would be the standard one. Using `brew config` for this is just waiting for trouble since that's not a stable API.

Ok, I can switch to that or figure something else out.

Reply:
> Oh, sorry, cross compile.

Brew doesn't cross-compile anything though. ARM builds run on ARM and x86 on x86.